.travis.yml

```diff
@@ -1,6 +1,11 @@
 language: go
 go:
-  - 1.8
+  - 1.8.x
+  - 1.9.x
+
 script:
   - make
   - make test
+
+notifications:
+  email: change
```

Comment:
What does this do?

Reply:
Only emails you if you break the build (change), rather than every time